testConnectivity.py

```diff
@@ -58,7 +58,7 @@ def getListOfBadTrigLinks(amcBoard,checkCSCTrigLink=False,debug=False,ohMask=0xf
 
     return listOfOHsWithBadTriggerLink
 
-def gbtCommIsGood(amcBoard, doReset=True, printSummary=True, ohMask=0xfff):
+def gbtCommIsGood(amcBoard, doReset=True, printSummary=True, ohMask=0xfff, gemType="ge11"):
```

Comment:
It doesn't seem like the argument `gemType` is actually used in this function.  Given that `amcBoard` is being passed as argument the type is already defined as a class member there.